chromiumcontent/build_libs.py

```diff
@@ -143,11 +143,12 @@ def gen_list(out, name, obj_dirs, exclude_dirs=[]):
         out,
         "obj_components",
         [
-            "components/appdu",
```

Comment:
Oh my.... I spent like a solid hour this afternoon wondering why this didn't work.... 😢